gensim/test/test_coherencemodel.py

```diff
@@ -77,6 +77,18 @@ def testCv(self):
         cm2 = CoherenceModel(topics=self.topics2, texts=texts, dictionary=dictionary, coherence='c_v')
         self.assertTrue(cm1.get_coherence() > cm2.get_coherence())
 
+    def testCuci(self):
+        """Test C_uci topic coherence algorithm on given topics"""
+        cm1 = CoherenceModel(topics=self.topics1, texts=texts, dictionary=dictionary, coherence='c_uci')
+        cm2 = CoherenceModel(topics=self.topics2, texts=texts, dictionary=dictionary, coherence='c_uci')
+        self.assertTrue(cm1.get_coherence() > cm2.get_coherence())
+
+    def testCnpmi(self):
+        """Test C_npmi topic coherence algorithm on given topics"""
+        cm1 = CoherenceModel(topics=self.topics1, texts=texts, dictionary=dictionary, coherence='c_npmi')
+        cm2 = CoherenceModel(topics=self.topics2, texts=texts, dictionary=dictionary, coherence='c_npmi')
+        self.assertTrue(cm1.get_coherence() > cm2.get_coherence())
+
     def testUMassLdaModel(self):
         """Perform sanity check to see if u_mass coherence works with LDA Model"""
         try:
```

Comment:
this test should be more than just checking for exception raised. can you do `self.assertTrue(lda1.get_coherence() > lda2.get_coherence())` here?